libensemble/generators.py

```diff
@@ -90,6 +91,18 @@ class LibensembleGenerator(Generator):
     ``ask_numpy/tell_numpy`` methods communicate numpy arrays containing the same data.
     """
 
+    def __init__(
+        self, History: npt.NDArray = [], persis_info: dict = {}, gen_specs: dict = {}, libE_info: dict = {}, **kwargs
+    ):
+        self.gen_specs = gen_specs
+        if len(kwargs) > 0:  # so user can specify gen-specific parameters as kwargs to constructor
+            self.gen_specs["user"] = kwargs
+        if not persis_info:
+            self.persis_info = add_unique_random_streams({}, 4, seed=4321)[1]
+            self.persis_info["nworkers"] = 4
```

Comment:
whats going on here

Reply:
The kwargs line is similar to https://github.com/Libensemble/libensemble/pull/1307#discussion_r1705983300

for `persis_info`, if libE ask/tell generators maintain the `H, persis_info, gen_specs, libE_info, **kwargs` signature, my thinking is users may prefer not needing to populate/maintain persis_info.

Its the circumstance where 

```python
aposmm = APOSMM(
    ub=1,
    lb=0,
)
```

is probably preferred to:

```python
aposmm = APOSMM(
    None,
    {},
    {
        "user": {"ub": 1, "lb": 0},
    },
    {}
)
```